public/app/plugins/datasource/elasticsearch/query_def.ts

```diff
@@ -151,6 +159,10 @@ export const pipelineOptions: any = {
     { text: 'predict', default: undefined },
     { text: 'minimize', default: false },
   ],
+  moving_fn: [
+    { text: 'window', default: 5 },
+    { text: 'script', default: 'MovingFunctions.unweightedAvg(values)' },
```

Comment:
I would remove this default and just give an hint as a placeholder in the field like we do with bucket_script:

![Screenshot 2020-10-20 at 16 38 54](https://user-images.githubusercontent.com/1170767/96609741-c156aa00-12f2-11eb-9ac2-ced97d57ec36.png)

maybe `eg. MovingFunctions.avg(values)` (just because it's shorter)
![Screenshot 2020-10-20 at 16 42 39](https://user-images.githubusercontent.com/1170767/96610223-48a41d80-12f3-11eb-8773-dc85ae1fe961.png)

This is to avoid users to rely too much on opinionated defaults. What do you think?

Reply:
Sounds good, I will make the change today.